tests/app.test.js

```diff
@@ -634,3 +634,99 @@ describe('InVEST subprocess testing', () => {
     spy.mockRestore();
   });
 });
+describe('Tab closing and switching', () => {
+  beforeAll(() => {
+    getInvestModelNames.mockResolvedValue(MOCK_INVEST_LIST);
+    getSpec.mockResolvedValue(SAMPLE_SPEC);
+    fetchValidation.mockResolvedValue(MOCK_VALIDATION_VALUE);
+    // mock out the whole UI config module
+    // brackets around spec.model_name turns it into a valid literal key
+    const mockUISpec = {[SAMPLE_SPEC.model_name]: {order: [Object.keys(SAMPLE_SPEC.args)]}};
+    jest.mock('../src/ui_config', () => mockUISpec);
+  });
+
+  afterAll(async () => {
+    jest.resetAllMocks();
+    jest.resetModules();
+  });
+
+  test('', async () => {
+    const { 
+      getByRole,
+      findByText, 
```

Comment:
I think these two queries aren't currently being used ... maybe they were left over from from earlier development?

Reply:
`getByRole` is used a couple times for getting the close button, but you're right `findByText` shouldn't be in there!